src/components/Avatar.js

```diff
@@ -5,5 +5,9 @@
 export default function Avatar(props) {    
     const { person } = props;
 
-    // return ...
+    return(
+        <div>
```

Comment:
no need to add divisions here - avoid creating unnecessary div's in your projects